docs/include/conf/cincinnati/visiting.rst

```diff
@@ -0,0 +1,69 @@
+Visiting Cincinnati
+===================
+
+Wondering where to stay and how to get around once you get into Cincinnati?
+
+The conference will be held at `Union Hall <http://unionhallcincy.com>`_ in the heart of Cincinnati's historic `Over-the-Rhine <https://en.wikipedia.org/wiki/Over-the-Rhine>`_ neighborhood.
+
+Eating
+------
+
+Snacks and drinks are provided throughout the conference. For meals during the conference, connect with new friends and explore one of Cincinnati's most diverse neighborhoods. Here are some resources that will help you find something good to eat:
+
+- The `Over-the-Rhine Chamber of Commerce <http://business.otrchamber.com/list/ql/restaurants-food-beverages-22?_ga=2.247727175.1927584494.1524751496-300914214.1524751496>`_ lists a number of restaurants and bars in the neighborhood
+- `Open Table <https://www.opentable.com/n/cincinnati/over-the-rhine-restaurants>`_ also lists great restaurants in Over-the-Rhine
+
+Where to stay
+-------------
+
+We are not working with any of the hotels for attendee accommodations, but there are many hotels available within walking distance of the venue:
+
+- `Symphony Hotel <http://www.symphonyhotel.com>`_
+- `The Cincinnatian Hotel <http://www.cincinnatianhotel.com>`_
+- `21c Museum Hotel <https://www.21cmuseumhotels.com/cincinnati/>`_
+- `Millennium Cincinnati <https://www.millenniumhotels.com/en/cincinnati/millennium-hotel-cincinnati/>`_
+- `Hilton Cincinnati Netherland Plaza <http://www3.hilton.com/en/hotels/ohio/hilton-cincinnati-netherland-plaza-CVGNPHF/index.html>`_
+
+Check TripAdvisor to find other hotels and accommodations in the Cincinnati area.
+
+How to get around
+-----------------
+
+Cincinnati is a small, walkable city with public transportation options available. Over-the-Rhine is about 1 mile north of the downtown area, which is bordered to the south by the Ohio River. We encourage you to connect with other documnentarians and explore during your stay.
+
+Streetcars and Buses
+~~~~~~~~~~~~~~~~~~~~
+
+Cincinnati has a `streetcar <http://www.cincinnatibellconnector.com>`_ that makes a loop from `Findlay Market <http://findlaymarket.org>`_ (north of Over-the-Rhine) to the `Great American Ball Park <https://www.mlb.com/reds/ballpark>`_ (downtown, on the Ohio River). You can also use Cincinnati's `bus network <http://www.go-metro.com/index.php>`_, which also connects to routes in northern Kentucky, just across the Ohio River.
```

Comment:
Worth mentioning the two closest stops. 12th and Vine if coming from downtown. Washington Park if coming from the north.

Reply:
Thank you again! I added this information.